backend/core/settings.py

```
@@ -0,0 +1,192 @@
+
+For the full list of settings and their values, see
+https://docs.djangoproject.com/en/5.1/ref/settings/
+"""
+
+from pathlib import Path
+import os
+from datetime import timedelta
+import environ
+import cloudinary
+import cloudinary.uploader
+import cloudinary.api
```

Ruff (0.8.2)

`cloudinary.api` imported but unused

Remove unused import: `cloudinary.api`

(F401)